<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilePickWidget</class>
 <widget class="QWidget" name="FilePickWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="text">
      <string>Path to file:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lePath"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowse">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFind">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnBrowse</sender>
   <signal>clicked()</signal>
   <receiver>FilePickWidget</receiver>
   <slot>browsePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFind</sender>
   <signal>clicked()</signal>
   <receiver>FilePickWidget</receiver>
   <slot>findPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browsePath()</slot>
  <slot>findPath()</slot>
 </slots>
</ui>
